pcdsdevices/mv_interface.py

```diff
@@ -64,6 +73,11 @@ class FltMvInterface(MvInterface):
 
     This lets us do more with the interface, such as relative moves.
     """
+    def __init__(self, *args, **kwargs):
+        if Presets._paths:
+            self.presets = Presets(self)
```

Comment:
I'm not sure I like the ordering that this implies. It means this attribute won't exist if we mess up the order of `setup_preset_paths`. Is there a way we can attach an empty `Presets` object that is updated by `setup_preset_paths`?

Reply:
Yeah, we can do this. I assumed there would be some use cases for not having a presets object sneaking into your session, so that was the motivation for the original implementation.